From efd37f21cf1e8ad4e86fb33303b84f7ca97c17b4 Mon Sep 17 00:00:00 2001
From: Flynn xu <flynn.xu@nxp.com>
Date: Tue, 6 Nov 2018 10:18:55 +0800
Subject: [PATCH 5028/5242] MLK-20173 ARM64:dts:imx8qm: update Xen verison
 number in imx8qm domU dts

commit  55ce109b2894880c72c5d1418c00e7b21aa2b976 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update Xen version number in imx8qm domu dts

Signed-off-by: Flynn xu <flynn.xu@nxp.com>
Reviewed-by: Peng Fan <peng.fan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts  |    2 +-
 .../dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts     |    2 +-
 .../boot/dts/freescale/fsl-imx8qm-mek-domu.dts     |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
index ab687ff1..0c6e515 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2-domu.dts
@@ -92,7 +92,7 @@
 	};
 
 	hypervisor {
-		compatible = "xen,xen-4.10", "xen,xen";
+		compatible = "xen,xen-4.11", "xen,xen";
 		reg = <0x0 0x38000000 0x0 0x1000000>;
 		interrupts = <GIC_PPI 15 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_LOW)>;
 		interrupt-parent = <&gic>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
index de51cad..fe2a649 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu-dpu1.dts
@@ -159,7 +159,7 @@
 	};
 
 	hypervisor {
-		compatible = "xen,xen-4.10", "xen,xen";
+		compatible = "xen,xen-4.11", "xen,xen";
 		reg = <0x0 0x38000000 0x0 0x1000000>;
 		interrupts = <GIC_PPI 15 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_LOW)>;
 		interrupt-parent = <&gic>;
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
index 23786b2..daccc8a 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek-domu.dts
@@ -164,7 +164,7 @@
 	};
 
 	hypervisor {
-		compatible = "xen,xen-4.10", "xen,xen";
+		compatible = "xen,xen-4.11", "xen,xen";
 		reg = <0x0 0x38000000 0x0 0x1000000>;
 		interrupts = <GIC_PPI 15 (GIC_CPU_MASK_SIMPLE(6) | IRQ_TYPE_LEVEL_LOW)>;
 		interrupt-parent = <&gic>;
-- 
1.7.9.5

